installer/roles/image_build/templates/Dockerfile.j2

```diff
@@ -209,7 +209,8 @@ RUN for dir in \
 # Adjust any remaining permissions
 RUN chmod u+s /usr/bin/bwrap ; \
     chgrp -R root ${COLLECTION_BASE} ; \
-    chmod -R g+rw ${COLLECTION_BASE}
+    chmod -R g+rw ${COLLECTION_BASE} ; \
+    chmod ugo+rx /usr/bin/launch_awx.sh /usr/bin/launch_awx_task.sh
```

Comment:
Why do you need both? The permissions should be respected if the files are already that permission.

Reply:
OK, if so, I'll force push dropping that part. I was thinking about the same, but as I was unsure about the build process, I put it to both. Thanks for simplifying this!

Reply:
I reverted this dockerfile change now.